backends-velox/src/test/scala/io/glutenproject/execution/VeloxStringFunctionsSuite.scala

```diff
@@ -43,82 +44,108 @@ class VeloxStringFunctionsSuite extends WholeStageTransformerSuite {
       .set("spark.sql.sources.useV1SourceList", "avro")
   }
 
+  def checkLengthAndPlan(df: DataFrame) {
+    assert(df.collect().length == 5)
```

Comment:
How about making the length and plan to be configurable and moving this function to `WholeStageTransformerSuite.scala` for reuse?  Eg. `checkLengthAndPlan(df: DataFrame, transformer: TransformSupport)`.